app/scripts/models/reliers/relier.js

```diff
@@ -76,6 +79,13 @@ define([
      */
     allowCachedCredentials: function () {
       return this.get('allowCachedCredentials');
+    },
+
+    /**
+     * Is the browser being automated? Set to true for selenium tests.
+     */
+    isAutomatedBrowser: function () {
```

Comment:
Perhaps this should be part of the broker. I'm not convinced this is "relier" info.